FlipperZero/Core/View/DesignPlaygroundView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  DesignPlaygroundView.swift
+//  Core
+//
+//  Created by Yachin Ilya on 25.03.2021.
+//
+
+import SwiftUI
+// swiftlint:disable closure_body_length indentation_width let_var_whitespace
+
+struct DesignPlaygroundView: View {
+    @State var fontSegment: Int = 0
+    @State var fontSize: Double = 32
+    private let minBound = 10
+    private let maxBound = 85
+    var body: some View {
+        VStack(alignment: .center) {
+            Picker("", selection: $fontSegment) {
+                Text("HelvetiPixel").tag(0)
+                Text("Born2bSportyV2").tag(1)
+                Text("Roboto-Regular").tag(2)
+            }
+            .pickerStyle(SegmentedPickerStyle())
+            .padding()
+            Slider(value: $fontSize,
+                   in: Double(minBound)...Double(maxBound),
+                   minimumValueLabel: Text("\(minBound) pt"),
+                   maximumValueLabel: Text("\(maxBound) pt")) {}
+                .padding(.horizontal)
```

Comment:
```suggestion
            Slider(
                value: $fontSize,
                in: Double(minBound)...Double(maxBound),
                minimumValueLabel: Text("\(minBound) pt"),
                maximumValueLabel: Text("\(maxBound) pt")) { }
                .padding(.horizontal)
```

Reply:
Don't like first arg 'hanging' on newline, but whatever.

Reply:
This way it's less brittle: when you change the function name, you don't need to update indentation for argument